.flowconfig

```diff
@@ -5,7 +5,7 @@
 [include]
 
 [libs]
-interface/
+src/flow/interfaces/
```

Comment:
Ohh, sorry.. Let me know if any other flow files must be moved somewhere else, since I don't have much experience with using them :)